src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -164,7 +165,16 @@ public void Start()
                 });
 
                 server.InboundNodeConnectionParameters = cloneParameters;
-                server.Listen();
+                try
+                {
+                    server.Listen();
+                }
+                catch (SocketException e)
+                {
+                    logger.LogCritical("P2P port " + listen.Endpoint.Port + " is being used. Please use a different port by running the node with '-port=[differentPort]'.");
```

Comment:
Please use string interpolation here $"P2P port {listen.Endpoint.Port} is bla..." 

also `this.logger`

no need to call ` logs.Append("Can't start listening on port: " + listen.Endpoint.Port);` you are throwing